build/gen_stub.php

```diff
@@ -3097,7 +3097,7 @@ public function getDeclaration(array $allConstInfos): string {
             $code .= $defaultValue->initializeZval($zvalName);
         }
 
-        [ $stringInit, $nameCode, $stringRelease ] = $this->getString( $propertyName );
+        [$stringInit, $nameCode, $stringRelease] = $this->getString( $propertyName );
```

Comment:
it's not just here, but everywhere in the PR.

Reply:
But I guess I should have also mentioned `parentheses` besides brackets ^^

Reply:
oh, duh